src/main/java/core/basesyntax/exceptions/WrongDateException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.exceptions;
+
+public class WrongDateException extends RuntimeException {
+    public WrongDateException(String message) {
```

Comment:
```suggestion
    public WrongDateException(String message, Throwable cause) {
```

for all your custom exeptions - include the original exception that was a reason of the custom one

Reply:
yes, makes sense